stdlib/builtins.pyi

```diff
@@ -1214,6 +1224,9 @@ class property:
     fset: Callable[[Any, Any], None] | None
     fdel: Callable[[Any], None] | None
     __isabstractmethod__: bool
+    if sys.version_info >= (3, 13):
+        __name__: str
```

Comment:
This isn't in https://docs.python.org/3.13/whatsnew/3.13.html, it probably should be.